app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -280,13 +281,26 @@ sealed class BubbleCta(
 
     override fun pixelShownParameters(): Map<String, String> = emptyMap()
 
-    class DaxFavoritesOnboardingCta() : BubbleCta(
+    class DaxFavoritesOnboardingCta : BubbleCta(
         CtaId.DAX_FAVORITES_ONBOARDING,
         R.string.daxFavoritesOnboardingCtaText,
         AppPixelName.FAVORITES_ONBOARDING_CTA_SHOWN,
         null,
         null
-    )
+    ) {
+        override fun showCta(view: View) {
+            super.showCta(view)
+            val accessibilityDelegate: View.AccessibilityDelegate =
+                object : View.AccessibilityDelegate() {
+                    override fun onInitializeAccessibilityNodeInfo(v: View?, info: AccessibilityNodeInfo) {
+                        super.onInitializeAccessibilityNodeInfo(v, info)
+                        info.text = v?.context?.getString(R.string.daxFavoritesOnboardingCtaContentDescription)
+                    }
+                }
+            // Using braille unicode inside textview, override description for accessibility
```

Comment:
Maybe worth clarifying _why_ we are doing that... e.g., using it to show something that looks like the overflow icon